# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Caches.h
# Commit: 40d2e0691861
# Full Hash: 40d2e069186171b8b1389e3388fad5927accbd7e
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 10 - Stop passing proto in a few places now that we can get it from the shape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106980
# ==============================================================================

diff -r ec72f104edbf -r 40d2e0691861 js/src/vm/Caches.h
--- a/js/src/vm/Caches.h	Sat Mar 06 01:05:22 2021 +0000
+++ b/js/src/vm/Caches.h	Sat Mar 06 01:05:23 2021 +0000
@@ -159,8 +159,8 @@
                          js::GlobalObject* global, gc::AllocKind kind,
                          NativeObject* obj);
 
-  /* Invalidate any entries which might produce an object with shape/proto. */
-  void invalidateEntriesForShape(Shape* shape, JSObject* proto);
+  /* Invalidate any entries which might produce an object with shape. */
+  void invalidateEntriesForShape(Shape* shape);
 
  private:
   EntryIndex makeIndex(const JSClass* clasp, gc::Cell* key,